src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationReader.cs

```diff
@@ -792,12 +792,11 @@ protected void ParseWsdlArrayType(XmlAttribute attr)
                 int colon = attr.Value.LastIndexOf(':');
                 if (colon < 0)
                 {
-                    attr.Value = _r.LookupNamespace("") + ":" + attr.Value;
+                    attr.Value = $"{_r.LookupNamespace("")}:{attr.Value}";
                 }
                 else
                 {
-                    attr.Value = string.Concat(_r.LookupNamespace(attr.Value.Substring(0, colon)), ":",
-                        attr.Value.AsSpan(colon + 1));
+                    attr.Value = $"{_r.LookupNamespace(attr.Value.Substring(0, colon))}:{attr.Value.AsSpan(colon + 1)}";
```

Comment:
```suggestion
                    attr.Value = $"{_r.LookupNamespace(attr.Value.AsSpan(0, colon))}:{attr.Value.AsSpan(colon + 1)}";
```

Reply:
span not valid argument;
string? LookupNamespace(string prefix);

Reply:
I thought we had a span-based overload, but I guess not.  You can ignore this suggestion then.